GOVERNANCE.md

```diff
@@ -0,0 +1,36 @@
+# Strimzi Governance
+
+This document defines project governance for the Strimzi project.
+
+## Voting
+
+The Strimzi project employs voting to ensure no single member can dominate the project. Any maintainer may cast a vote.
+
+For formal votes, a specific statement of what is being voted on should be added to the relevant github issue or PR.
+Maintainers should indicate their yes/no vote on that issue or PR, and after a suitable period of time, the votes will be tallied and the outcome noted.
+
+## Changes in Maintainership
+
+New maintainers are proposed by an existing maintainer and are elected by a ⅔ majority vote.
+
+Maintainers can be removed by a ⅔ majority vote.
+
+## Approving PRs
+
+PRs may be merged after receiving at least two positive votes. If the PR author is a maintainer, this counts as a vote.
+
+## Github Project Administration
+
+Maintainers will be added to the collaborators list of the Strimzi repository with "Write" access.
+
+After 6 months a maintainer will be given "Admin" access to the Strimzi repository.
```

Comment:
Going back to the discussion on Sync-up ... is the maintainership a thing or repo or organization?

```suggestion
After 6 months a maintainer will be given "Admin" access to the Strimzi GitHub organization.
```